packages/liburcu.rb

```diff
@@ -1,38 +1,23 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Liburcu < Package
+class Liburcu < Autotools
   description 'Userspace RCU (read-copy-update) library.'
   homepage 'https://liburcu.org/'
-  version '0.14.0-1'
+  version '0.14.0-gcc14'
```

Comment:
Why are we tagging it with the gcc version here?

Reply:
@satmandu?

Reply:
I assume this is to indicate the gcc version that compiled the binaries.

Reply:
I would have thought that is only nessecary in certain cases, like things interacting with the compiler itself.

Reply:
The associated libraries are built with the gcc version embedded also.  Below is what can result from different gcc versions.
```
lto1: fatal error: bytecode stream in file '/usr/local/lib64/liburcu.a' generated with LTO version 13.0 instead of the expected 14.0
```

Reply:
Sorry to keep moving the goalposts :sweat_smile:, but would building shared libraries instead solve this?

Reply:
Wait, why isn't liburcu being built with shared libraries!

Good catch!

Reply:
I removed the static libraries and rebuilt the binaries.  Thank you for the feedback.